client/web/src/enterprise/batches/detail/changesets/BatchChangeChangesets.tsx

```diff
@@ -175,8 +226,14 @@ export const BatchChangeChangesets: React.FunctionComponent<Props> = ({
                     location={location}
                     useURLQuery={true}
                     listComponent="div"
-                    listClassName="batch-change-changesets__grid mb-3"
-                    headComponent={BatchChangeChangesetsHeader}
+                    listClassName={
+                        enableSelect
+                            ? 'batch-change-changesets__grid-with-checkboxes mb-3'
```

Comment:
I think BEM would be

```suggestion
                            ? 'batch-change-changesets__grid--with-checkboxes mb-3'
```